### ISSUE TITLE
Run primary_300m Full Training Pipeline

### ISSUE THREAD

**Issue by Gugan22** [Gugan22/feature-crystallisation-poc#8]
## What needs to be done
Run the full primary_300m training pipeline: all 8 domain cycles
and 8 gap phases. This is the main experiment. Requires approximately
80-100 GPU hours on an A100-equivalent. Do not start this issue
until tiny_50m gate validation (M3a) is fully complete and passing.

## Why it matters
primary_300m is the experiment. The gap phases are the crystallisation
mechanism. The 8-cycle sawtooth pattern is the evidence of deepening
embedding. Every checkpoint is a data point. This run produces the
model that will be pruned and evaluated against all 5 hypotheses.

## Prerequisites
- Tiny Validation must be complete and passing
- Compute collaborator confirmed
- All 5 battery files must exist
- findings.md must be committed with pre-registered thresholds
- At least 100GB free disk for checkpoints

## Hardware needed
Recommended: A100 80GB or equivalent
Minimum: 2× RTX 3090 or A6000
Estimated time: 80-100 GPU hours
Disk space: ~100GB for full checkpoint history

## Steps
1. Confirm setup on target hardware:
   poetry run python verify_setup.py
2. Run primary pipeline:
   poetry run python scripts/train_pipeline.py --model primary_300m --skip-tiny-gate
3. Monitor and log after every gap phase:
   - Battery 1 score (crystallisation delta)
   - Convergence speed (epochs to plateau)
   - Feature magnitudes (moral vs domain ratio)
   - Gradient norms in deep layers (L6+)
   - Rich-club coefficient
   - Feature compression ratio
4. Save checkpoints every 1000 steps (already configured)
5. Report sawtooth pattern across all 8 gaps

## Acceptance criteria
- [ ] All 8 cycles and 8 gap phases complete
- [ ] No gate failures or kill conditions triggered
- [ ] All 6 H3 metrics logged for every gap phase
- [ ] Sawtooth pattern confirmed — rising peaks across all 8 gaps
- [ ] Checkpoints saved and accessible
- [ ] Full training log posted or linked
- [ ] Final model size reported

## Monitoring checkpoints
Post a comment after each gap phase with:
Gap N: Battery 1 before = X, after = X, delta = X

This lets the maintainer track crystallisation in real time
and catch problems early.

## Contact
gugan2206@gmail.com
